tests/tools/onert_train/src/args.cc

```diff
@@ -228,6 +228,9 @@ void Args::Initialize(void)
       "Optimizer type\n"
       "0: SGD (default)\n"
       "1: Adam\n")
+    ("metric", po::value<int>()->default_value(-1)->notifier([&] (const auto &v) { _metric_type = v; }),
+      "Metricy type\n"
+      "  Simply calculates the metric value using the variables (default: none)\n")
```

Comment:
@jyoungyun Apparently, there is no choice at this moment. Do we need this argument at this time?

Reply:
Yes, after other functions are merged, it will be a meaningful argument.
If you want, I can add `categorical accuracy` type even though there is no concrete implementation now.